CHANGES

```diff
@@ -3,6 +3,9 @@ OpenVDB Version History
 
 Version 7.0.1 - In Development
 
+    Bug fixes:
+    - Fixed a bug where empty grids were still returning true from
+      evalActiveVoxelBoundingBox when leaf nodes were still present.
```

Comment:
This description was confusing to me since an empty grid (as defined by the Grid::empty() = Tree::empty()) is a Tree without any nodes. However, this method computes the bounding box of active values (irrespective of where they live in the tree). May I therefore suggest this wording instead:

- Fixed a bug where grids with no active values might return true when the method evalActiveVoxelBoundingBox is called. The correct behavior is to only return true if the grid contains any active values.

Reply:
True, this confusion is precisely how the issue arose :>

In my mind, a tree with no active values *is* empty.  It is a purely internal matter whether there any leaf nodes allocated or not.   This viewpoint is likely why I also don't like "densify" functions as they are meaningless to me :>

I'll try and clarify the wording as you suggest.

Reply:
thanks